### ISSUE TITLE
Code Quality and Coherence

### ISSUE THREAD

**Issue by wedusk101** [wedusk101/NumericalAnalysis#3]
I went through some of the code and from what I see there is a pretty serious problem with the code
quality and readability. I had uploaded the MODULES file to work as a guideline for the code organization.
The variable names don't make sense and they are not at all documented. The methods had to be static.
Rohit added his files as a separate package. Those packages import another package MyIO which is adding unwanted dependency. **We just need methods that work together after which we will test the modules and integrate them into one SINGLE package.** Please stick to the guidelines folks.

**Comment by ImRohitSingh** [comment]
I have answered your question beneath the updated code! Do see it! Forward interpolation methods are not to be used anywhere but in Backward. For which I have requested Vishal to create an object of the Forward class and call the methods differencesCal() [method to calculate differences] and showDifferences() [method to show the differences in table form] in his code. Rest all other Interpolation procedures requires Central Differences. Hence these differences are needless to them. Moreover they are not a part of this project even. So making them as 'static' would be as complex as using the MyIO package. As you have asked, I have commented my code. Do contradict for any ambiguity.
